src/Tizen.Uix.InputMethod/Tizen.Uix.InputMethod/InputMethodEditor.cs

```diff
@@ -2139,5 +2159,55 @@ public static void SetFloatingDragEnd()
                 throw InputMethodExceptionFactory.CreateException(error);
             }
         }
+
+        /// <summary>
+        /// Updates the state of input panel event.
+        /// </summary>
+        /// <privilege>
+        /// http://tizen.org/privilege/ime
+        /// </privilege>
+        /// <param name="type">The input panel event type.</param>
+        /// <param name="value">The value of event type.</param>
+        /// <exception cref="InvalidOperationException">
+        /// This can occur due to the following reasons:
+        /// 1) The application does not have the privilege to call this function.
+        /// 2) The IME main loop has not yet started.
+        /// </exception>
+        /// <since_tizen> 5.5 </since_tizen>
+        public static void UpdateInputPanelEvent(EventType type, uint value)
```

Comment:
1. Events that occur in IME.
    1) Language = When IME's language is changed
    2) ShiftMode = When Shift key is pressed
    3) Geometry = When IME size or size is changed
2. This API for forwarding events to widget. I will change 'Update' to 'Send'.

Reply:
> Language = When IME's language is changed
in this case, What mean the `uint value`? which value can be passed?
> ShiftMode = When Shift key is pressed
in this case, What mean the `uint value`? which value can be passed?
> EventType == Geometry
in this case, What mean the `uint value`? which value can be passed?

How about split into each method?
``` c#
SendLanguageUpdated(Language language);
SendShiftModeUpdated(bool enable);
SendGeometryUpdated(int x, int y, int width, int height);
```

Reply:
Only 0 and 1 exist because Because it only deliver that the state of the IME has changed. But use uint because other events may be added to the enum. And I think the current method is more flexible in dealing with new requirements.

Reply:
0과 1이 의미하는것이 무엇입니까?
API를 모호하게 정의하지 말고 좀 더 명확하게 정의하시고, 확장이 필요한 경우 overloading 방법을 사용하여 다른 API로 확장하는것을 권장합니다.

Reply:
추가된 API는 변경되었다는 정보만 전달해주는 것이라 현재는 크게 의미가 없습니다. 현재 C API의 구조는 IME의 해당 이벤트가 변경되었다는 정보만 전달하면 위젯에서 변경된 값을 얻어가고 있습니다. 추가하려는 API의 value parameter가 모호하기 때문에 아래와 같이 수정하도록 하겠습니다.

NotifyLanguageUpdated();
SendShiftModeUpdated(ShiftMode mode);
SendGeometryUpdated(int x, int y, int width, int height);